releasenotes/notes/0.2/0_2_release-eef5e3ba256fc750.yaml

```diff
@@ -0,0 +1,38 @@
+---
+prelude: >
+    The Qiskit Experiments 2.0 release includes several bug fixes and
+    improvements and many new experiments added to the
+    :mod:`~qiskit_experiments.library`. Added experiments include a full suite
+    of single-qubit gate calibration and characterization experiments, and
+    two-qubit Cross-resonance Hamiltonian characterization experiments.
+
+    There are also numerous changes and improvements to the base classes in
+    :mod:`~qiskit_experiments.framework` and
+    :mod:`~qiskit_experiments.calibration_management` to make
+    developing new experiments easier, to improve JSON serialization, and to 
+    improve storing and loading experiments for the IBM experiment database
+    service.
```

Comment:
Probably we need to write the experiment data entries generated with v0.1 cannot be loaded due to class structure change? (and we don't plan to provide backport for such early versions)

Reply:
Is this something you could test? I think some experiments from 0.1 can be loaded (but with some json serialization warnings) while others will probably raise exceptions. I can add a general warning but if we know for sure which ones will fail I can add that.